tests/53-sim-binary_tree.py

```diff
@@ -36,11 +77,15 @@ def test(args):
     f.add_arch(Arch("x86"))
     f.set_attr(Attr.CTL_OPTIMIZE, 2)
 
-    for i in range(330):
-        if (i == 10 or i == 53 or i == 61 or i == 255):
-            f.add_rule(ERRNO(i), i, Arg(0, EQ, i))
+    for entry in table:
+        print(entry)
+        if entry["syscall"] == "mprotect" or \
+           entry["syscall"] == "dup" or \
+           entry["syscall"] == "alarm":
```

Comment:
My comments from the C variant of this test still apply here although the implementation will obviously be slightly different.